snekbox/nsjail.py

```diff
@@ -171,12 +171,15 @@ def _consume_stdout(nsjail: subprocess.Popen) -> str:
 
         return "".join(output)
 
-    def python3(self, code: str, *args) -> CompletedProcess:
+    def python3(self, code: str, *args, extra_args: Iterable[str] = ("-c",)) -> CompletedProcess:
```

Comment:
The naming is currently too ambiguous. I think `*args` should be `nsjail_args` (no longer varargs) and `extra_args` should be `py_args`.

Reply:
Please add a test for the Python arguments. Now that I think about it, the nsjail args also lack a test, so feel free to add one for them too.

In this case, it may be better to mock the `subprocess.run` call and check the arguments it's called with rather than checking the result of the evaluation, unless you have a better idea.